ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/middleware/cimi_params_impl.clj

```diff
@@ -115,12 +115,14 @@
   then an exception is thrown."
   [{:keys [params] :or {:params {}} :as req}]
   (if-let [filter-param (get params "$filter")]
-    (->> filter-param
-         (as-vector)
-         (wrap-join-with-and)
-         (parser/parse-cimi-filter)
-         (throw-illegal-for-invalid-filter)
-         (add-cimi-param req :filter))
+
+    (add-cimi-param req :filter filter-param)
```

Comment:
This change seems to be causing all of the tests in `cimi_params_impl_test.clj` using `process-filter` to fail.  Before this was returning the parsed filter, now this is returning the request with the raw, unparsed filter.  Which should be the correct behavior?